tests/test_github_actions_integration.py

```
@@ -157,28 +168,51 @@ def xfail_known_act_runtime_limitations(logs: str) -> None:
         )
 
 
-def assert_act_result(project: CopierProject, code: int, logs: str) -> None:
+def assert_act_result(
+    project: CopierProject,
```

Ruff (0.16.0)

[warning] Unused function argument: `project`

(ARG001)